setup.py

```diff
@@ -210,18 +210,6 @@ def extract_version():
         },
     data_files=[('iris', ['CHANGES', 'COPYING', 'COPYING.LESSER'])],
     tests_require=['nose'],
-    features={
-        'unpack': setuptools.Feature(
-            "use of UKMO unpack library",
-            standard=False,
-            ext_modules=[
-                setuptools.Extension(
-                    libraries=['mo_unpack'],
-                    include_dirs=[np.get_include()]
-                )
-            ]
-        )
-    },
     cmdclass={'test': SetupTestRunner, 'build_py': BuildPyWithExtras,
```

Comment:
@lbdreyer I take it this is only used for pp-packing?  Nothing else will be affected by the removal of this code section?